src/main/java/it/gov/pagopa/arc/exception/ArcExceptionHandler.java

```diff
@@ -23,11 +23,6 @@ public ResponseEntity<ErrorDTO> handleBizEventsInvocationException(RuntimeExcept
         return handleArcErrorException(ex, request, HttpStatus.INTERNAL_SERVER_ERROR, ErrorDTO.ErrorEnum.GENERIC_ERROR);
     }
 
-    @ExceptionHandler(BizEventsReceiptNotFoundException.class)
-    public ResponseEntity<ErrorDTO> handleBizEventsReceiptNotFoundException(RuntimeException ex, HttpServletRequest request){
-        return handleArcErrorException(ex, request, HttpStatus.NOT_FOUND, ErrorDTO.ErrorEnum.RECEIPT_NOT_FOUND_ERROR);
-    }
```

Comment:
It is used by the /notices/{eventId}/receipt API, do not remove it.

Reply:
restored